include/internal/catch_test_spec.h

```diff
@@ -22,10 +22,16 @@
 
 namespace Catch {
 
+	struct PatternDescription {
+		std::string name;
+		std::string pattern;
+	};
+
     class TestSpec {
         struct Pattern {
             virtual ~Pattern();
-            virtual bool matches( TestCaseInfo const& testCase ) const = 0;
+			virtual bool matches(TestCaseInfo const& testCase) const = 0;
```

Comment:
We use spaces, not tabs.

Reply:
Fresh installation of VS, me too. I'll untabify it.